.gitignore

```diff
@@ -1,3 +1,4 @@
+Gemfile.lock
```

Comment:
Don't check in Gemfile.lock for gems.

http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/